proto/provenance/metadata/v1/tx.proto

```diff
@@ -47,9 +48,15 @@ service Msg {
   rpc AddRecordSpecification(MsgAddRecordSpecificationRequest) returns (MsgAddRecordSpecificationResponse);
   // DeleteRecordSpecification deletes a record specification
   rpc DeleteRecordSpecification(MsgDeleteRecordSpecificationRequest) returns (MsgDeleteRecordSpecificationResponse);
+
+  // P8EMemorializeContract records the results of a P8e contract execution as a session and set of records in a scope
+  // It only exists to help facilitate the transition. The MemorializeContract is the one to use now.
+  rpc P8eMemorializeContract(MsgP8eMemorializeContractRequest) returns (MsgP8eMemorializeContractResponse) {
+    option deprecated = true;
+  };
 }
 
-// MsgMemorializeContractRequest is a request from a P8e execution environment to record results of a contract
+// MsgMemorializeContractRequest is a request from an execution environment to record results of a contract
```

Comment:
This function is a P8e only  concept ... it should remain tagged as P8e (or more likely) removed entirely